.github/ISSUE_TEMPLATE/ENHANCEMENT.md

```diff
@@ -0,0 +1,43 @@
+---
+name: '📈 Enhancement'
+about: Enhancement to our codebase that isn't a adding or changing a feature
+---
+
+## Overview
+
+...
+
+## Motivation
+
+> What inspired this enhancement? What makes you think this should be in `quilt`?
+
+...
+
+## Labels
+
+- [ ] Add the `Type: Enhancement` label to this issue.
+
+### Area
+
+- [ ] Add any relevant `Area: *` labels to this issue
+
+### Scope
+
+- Is this issue related to a specific package?
+
+  - [ ] Tag it with the `Package: *` label.
+
+### Effort
+
+Do have an idea of how much effort will be required?
+
+- [ ] please tag it with the appropriate `Effort: *` label.
+
+- [ ] Please delete the labels section before submitting your issue
+
+---
+
+## Checklist
+
+- [ ] I have added the appropriate labels to this issue
```

Comment:
I think this is redundant, the previous items are checklists that imply the same thing.